### DISCUSSION TITLE
Support empty titles & add an option to not auto-fill it

### DISCUSSION THREAD

**Discussion by parasiteoflife** [gotson/komga#1572] [Ideas]
### Describe your suggested feature

Currently Komga uses the filenames as titles even when files include a `ComicInfo.xml` file inside that doesn't have a title tag. What I'm asking is for Komga not to require titles to have a value, as there are several comic series whose issues don't have titles, and an option to not auto fill the title with the filename when the file includes a `ComicInfo.xml` file.

### Other details

I think that if someone uses a ComicInfo.xml file and the file doesn't include certain tags then no assumptions about them should be made, if I don't include a title tag Komga shouldn't assume and use the filename, if I don't include it is because either I don't want the issue to have one or the issue itself doesn't have one. It's OK to assume information about the series because currently there is no way to specify it in any sidecar files, but the same doesn't apply to issues.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
> when files include a `ComicInfo.xml` file inside that doesn't have a title tag

no tag doesn't mean no title, but that the title is not present, which is different

> as there are several comic series whose issues don't have titles

can you provide some examples?

> if I don't include a title tag Komga shouldn't assume and use the filename

that's not how it works. Metadata is additive, by default the title is the filename, then metadata is loaded from `Comicinfo.xml`, but if no title tag is present, then no information can be used.

**Comment by parasiteoflife** [comment]
> but if no title tag is present, then no information can be used.

Yes, please, this would be great.

> can you provide some examples?

From the last entries I've added:

https://www.tebeosfera.com/colecciones/western_1982_valenciana.html
https://comicvine.gamespot.com/star-trek-the-next-generationx-men-second-contact/4050-64021/
https://comicvine.gamespot.com/paradise-x/4050-9547/
https://comicvine.gamespot.com/star-trek/4050-145706/
